components/image/index.tsx

```diff
@@ -1,11 +1,31 @@
 import { MediaPlaceholder, InspectorControls } from '@wordpress/block-editor';
 import { Spinner, FocalPointPicker, PanelBody, Placeholder } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import PropTypes from 'prop-types';
+import React from 'react';
+import type { FC } from 'react';
 
 import { useMedia } from '../../hooks/use-media';
 
-const Image = (props) => {
+type FocalPoint = {
+	x: number,
+	y: number,
+};
+
+type ImageProps = {
+	id: number;
+	size: string;
+	onSelect(value: { id: number } & { [k: string]: any }): void;
```

Comment:
@fabiankaegy  I would stick with `key` instead of `x` or `k` whenever possible, just to follow a pattern. 😉